drivers/mmc/Makefile

```diff
@@ -74,6 +74,7 @@ obj-$(CONFIG_MMC_SDHCI_TANGIER)		+= tangier_sdhci.o
 obj-$(CONFIG_MMC_SDHCI_TEGRA)		+= tegra_mmc.o
 obj-$(CONFIG_MMC_SDHCI_XENON)		+= xenon_sdhci.o
 obj-$(CONFIG_MMC_SDHCI_ZYNQ)		+= zynq_sdhci.o
+obj-$(CONFIG_MMC_SDHCI_CVITEK)		+= cv180x_sdhci.o
```

Comment:
We'd better put the code according to the alphabetical order, that is

```makefile
obj-$(CONFIG_MMC_SDHCI_BVITEK)		+= bv180x_sdhci.o
obj-$(CONFIG_MMC_SDHCI_CVITEK)		+= cv180x_sdhci.o
obj-$(CONFIG_MMC_SDHCI_DVITEK)		+= dv180x_sdhci.o
```

Reply:
let's leave it as is for now, we'd refactor this when pushing to upstream